### PR TITLE
Feature/target tracking, landmark msg & action

### LINKED ISSUE #2: [TASK] Landmark message
### Description of task

Create a landmark message as described in the Landmark Server V2 task: https://github.com/vortexntnu/vortex-asv/issues/114

### Suggested Workflow

_No response_

### Specifications

_No response_

### Contacts

_No response_

### Code Quality

- [ ] Every function in header files are documented (inputs/returns/exceptions)
- [ ] The project has automated tests that cover MOST of the functions and branches in functions (pytest/gtest)
- [ ] The code is documented on the wiki (provide link)

### LINKED ISSUE #3: [TASK] Create action for the landmark server
### Description of task

Create an action as described in https://github.com/vortexntnu/vortex-asv/issues/114

### PR DESCRIPTION
**Pull request by jorgenfj**
Landmark msg and action for use in the landmark server
Also includes some msgs for the target tracking visualization. I am unsure of whether they are used or should be deleted.
Closes #3 
Closes #2

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,25 +1,46 @@
 cmake_minimum_required(VERSION 3.5)
 project(vortex_msgs)
 
+# include_directories(
+#   ${CMAKE_CURRENT_SOURCE_DIR}/msg
+# )
+
```
Comment (alekskl01) [inline CMakeLists.txt:7]: Remove if not used

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,25 +1,46 @@
...
   "msg/KMBinary.msg"
+  "msg/Position2DWithCovariance.msg"
+  "msg/PosVel2DWithCovariance.msg"
+  "msg/VisualizationData.msg"
+  "msg/VisualizationDataArray.msg"
+  "msg/Landmark.msg"
+  "msg/LandmarkArray.msg"
+  "msg/OdometryArray.msg"
+  "msg/Parameter.msg"
+  "msg/ParameterArray.msg"
```
Comment (alekskl01) [inline CMakeLists.txt:28]: Remember to remove the unused messages here aswell.

File: msg/Parameter.msg [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Ros parameter message
+
+# Name of the parameter
+string name
+
+# Value of the parameter
+string value
+
+# Type of the parameter
+string type
```
Comment (alekskl01) [inline msg/Parameter.msg:10]: What is the purpose of this message? Do we need a message for ROS parameters?

**Review by alekskl01 — CHANGES_REQUESTED** [review]

**Comment by TristanWolfram** [comment]
I removed the messages in CMakeLists.txt as well. Also removed the comment since it is not used.
I don't know anything about the Parameter message. So, @jorgenfj du you know, if we need this?

**Review by alekskl01 — APPROVED** [review]

File: msg/Parameter.msg (hunk 3 above)
Comment (jorgenfj) [reply msg/Parameter.msg:10]: It is not necessary. Can be solved through param event callbacks. Will discuss with @etfroeland when he comes back. But should be kept for now to visualize results from easter testing

**Pull request merged**